2-With-broker/README.md

```diff
@@ -327,21 +329,31 @@ If you are using the system browser for interactive authntication, is possible t
 ```
 //NOTE: the slash before your signature value added to the path attribute
 //This uses the base64 encoded signature produced above.
-<intent-filter>
-      <data android:scheme="msauth"
-                    android:host="Package Name"
+      <intent-filter>
+        <action android:name="android.intent.action.VIEW" />
+        <category android:name="android.intent.category.DEFAULT" />
+        <category android:name="android.intent.category.BROWSABLE" />
+        <data android:scheme="msal"Client ID"" android:host="auth" />
+        <data android:scheme="msauth"
```

Comment:
we already ask customers to add an intent filter to handle the system browser . however, when broker is used the redirect URI is different so we ended up asking the customer to add a second intent filter to handle the broker redirect URI as well. the issue here is that the intent filters can ovveride each other based on the order and break the functionality entirely. the solution is to combine the intent filters as shown here.

Reply:
I think these changes should not only be documented, but be made part of the sample.

Reply:
I agree!

Reply:
The issue here is that the changes require a specific signature from the broker redirect URI that is generated at build time. Unfortunately we do not know what that will be so I would prefer not to add this entry to the sample with a blank signature. I wish there was a better way of handling it but I'm not sure what we can do here. Any other suggestions to improve this?

@jmprieur @bgavrilMS

Reply:
Let's configure the embedded browser fallback and add a link to the docs for people wanting to use the system browser fallback?